packages/indexer-common/src/indexer-management/resolvers/actions.ts

```diff
@@ -24,9 +24,22 @@ async function executeQueueOperation(
   logger: Logger,
   action: ActionInput,
   actionsAwaitingExecution: Action[],
+  recentlyFailedActions: Action[],
   models: IndexerManagementModels,
   transaction: Transaction,
 ): Promise<ActionResult[]> {
+  const recentlyFailedAction = recentlyFailedActions.filter(
+    (a) => a.deploymentID === action.deploymentID && a.type === action.type,
```

Comment:
I think we are going to want to be a bit more strict with checking for a match here. 

If an indexer made some changes the action that is attempting to be added to the queue could be different than the recently failed action and thus not expected to fail. 

It'll be a bit different depending on the action type: 
- ActionType.ALLOCATE - Also check `allocationAmount`
- ActionType.UNALLOCATE - Also check `poi` and `force`
- ActionType.REALLOCATE - Also check `poi`, `force`, and `allocationAmount`